hols/oci-basic-setup/README.md

```diff
@@ -0,0 +1,203 @@
+   unzip helidon-cli.zip
+   ```
+3. Make sure that JDK 21 exists in the path.
+4. Execute the cli to generate a Helidon Microprofile application project.
+   ```shell
+   ~/helidon-3.0.6/bin/helidon init
+   ```
+5. When it prompts for `Helidon version`, choose the default (or the latest Helidon 4 version)
+   ```shell
+   Helidon versions
+   (1) 4.2.1
+   (2) 3.2.12
+   (3) 2.6.11
+   (4) Show all versions
+   Enter selection (default: 1):
+   ```
+6. When prompted to `Select a Flavor`, choose `Helidon MP`
+   ```shell
+   | Helidon Flavor
+   
+   Select a Flavor
+     (1) se | Helidon SE
+     (2) mp | Helidon MP
+   Enter selection (default: 1): 2
+   ```
+7. When prompted to `Select an Application Type`, choose `oci`  
+   ```shell
+   Select an Application Type
+   (1) quickstart | Quickstart
+   (2) database   | Database
+   (3) custom     | Custom
+   (4) oci        | OCI
+   Enter selection (default: 1): 4
+   ```
+8. When prompted for `Project groupId`, `Project artifactId`, `Project version` and `Java package name`, just accept the default values. Once completed, this will generate an `oci-mp` project.
+10. Go to the `oci-mp` directory.
+    ```shell
+    cd ~/oci-mp
+    ```
+11. Run the `update_config_values.sh` utility script from the `oci-basic-setup` local repository to update the config parameters. Specify the directory location of the Helidon application as an argument to this script.
+    ```shell
+    ~/helidon-labs/hols/oci-basic-setup/update_config_values.sh ~/oci-mp
+    ```
+    Invoking this script performs the following:
+    1. Updates the `~/oci-mp/server/src/main/resources/application.yaml` config file to set up a Helidon feature that sends Helidon generated metrics to the OCI monitoring service.
+       1. compartmentId - The Compartment OCID used for this demo.
+       2. namespace - This can be any string but for this demo, will be set to `helidon_metrics`.
+    2. Updates the `~/oci-mp/server/src/main/resources/META-INF/microprofile-config.properties` config file to set up configuration parameters used by the Helidon app code to perform integration with OCI Logging and Metrics service.
+       1. oci.monitoring.compartmentId - Compartment ocid that is used for this demo
+       2. oci.monitoring.namespace - This can be any string but for this demo, this will be set to `helidon_application`.
+       3. oci.logging.id - Application log id that was provisioned by the terraform scripts.
+
+    **Note:** Make sure to validate that `application.yaml` and `microprofile-config.properties` were updated by checking that the mentioned config parameters were properly populated.
+12. To prepare for the build, ensure that JDK 21 and Maven 3.8+ exist and are set in the PATH environment variable of the Cloud Shell terminal.
+13. Build the application.  
+    ```shell
+    mvn clean package
+    ```
+
+### Deploy the Helidon OCI application:
```

Comment:
Deploy the Helidon OCI Application to your compute instance

Reply:
Updated